views/list-job-roles.html

```diff
@@ -19,7 +19,9 @@ <h2 style="color:red;font-weight:bold;">{{ errormessage }}</h2>
     </tr>
     {% for role in roles %}
     <tr>
-        <td>{{ role.roleName }}</td>
+        <td>
+            <a href="/job-roles/{{role.roleId}}" target="_blank" id="roleNameInList">{{ role.roleName }}</a>
+        </td>
```

Comment:
how does this look - would be good to add a screenshot of your changes in the UI to the PR